.gitlab-ci.yml

```diff
@@ -71,15 +78,36 @@ check-line-width:
   allow_failure:                   false
 
 
-test-linux-stable:                 &test
+cargo-audit:
+  stage:                           test
+  <<:                              *compiler_info
+  only:                            *not-prs
```

Comment:
syntax inconsistent. is it working both ways?

Reply:
if it's set like this:
```
.not-prs:
  only:                            &not-prs # alias inside the job
```
it can be only called like this:
```
only:                            *not-prs
```

But if it's set this way:
```
.not-prs:                      &not-prs # alias for the job name
  only:                            
```
it can be only called like this:
```
<<: *not-prs
```
I thought it would be a bit more transparent to do it this way, but do not insist on it.

Reply:
no looks good, so it could be used for compiler-info as well? - I also don't insist, okay with both.